File-Formats/XlsIO/faqs/how-to-detect-merged-cells-in-Excel.md

```diff
@@ -0,0 +1,179 @@
+---
+title: How to detect merged cells in Excel | XlsIO | Syncfusion
+description: This page explains how to detect merged cells in an Excel document using Syncfusion .NET Excel library (XlsIO).
+platform: file-formats
+control: XlsIO
+documentation: UG
+---
+
+# How to detect merged cells in Excel?
+
+The merged cells in an Excel worksheet can be detected through [MergedCells](https://help.syncfusion.com/cr/file-formats/Syncfusion.XlsIO.IWorksheet.html#Syncfusion_XlsIO_IWorksheet_MergedCells) of IWorksheet. The following complete code snippet explains this.
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C#" %}
+using (ExcelEngine excelEngine = new ExcelEngine())
+{
+  IApplication application = excelEngine.Excel;
+  application.DefaultVersion = ExcelVersion.Xlsx;
+  IWorkbook workbook = application.Workbooks.Open("Sample.xlsx");
+  IWorksheet worksheet = workbook.Worksheets[0];
+
+  //Create a new worksheet
+  IWorksheet mergedCells = workbook.Worksheets.Create("MergedCells");
+
+  //Get the list of merged cells
+  for (int i = 0; i < worksheet.MergedCells.Length; i++)
+  {
+    mergedCells.Range["A" + (i+1).ToString()].Text = (i+1).ToString()+"th Merged region = "+ worksheet.MergedCells[i].Address;
+  }
+
+  //Autofit the used range
+  mergedCells.UsedRange.AutofitColumns();
+
+  workbook.SaveAs("Output.xlsx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+Using excelEngine As ExcelEngine = New ExcelEngine()
+  Dim application As IApplication = ExcelEngine.Excel
+  application.DefaultVersion = ExcelVersion.Xlsx
+  Dim workbook As IWorkbook = application.Workbooks.Open("Sample.xlsx")
+  Dim worksheet As IWorksheet = workbook.Worksheets(0)
+
+  'Create a new worksheet
+  Dim mergedCells As IWorksheet = workbook.Worksheets.Create("MergedCells")
+
+  'Get the list of merged cells
+  For i As Integer = 0 To worksheet.MergedCells.Length - 1 Step 1
```

Comment:
@KonduruKeerthi  - We should avoid using variable name as "i","j" and other characters. Instead, we should provide meaning names such as "pos", "cellIndex" and etc.

Reply:
@Mohan2401 - I have made changes as suggested.